<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QPushButton" name="btEnvoi" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>300</y>
      <width>106</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Search</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_mess" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionQuitter" >
   <property name="text" >
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionJoin" >
   <property name="text" >
    <string>join</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
